.cilibs/compile_code.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+GATE_VERSION=release-1.20.x
+BUILD_OS=linux
+BUILD_ARCH=amd64
+
+
+while getopts "o:a:g:" opt; do
+  case ${opt} in
+    o) #Build OS
+      BUILD_OS=${OPTARG}
+      ;;
+    a) #Build arch
+      BUILD_ARCH=${OPTARG}
+      ;;
+    g) #Gate version
+      GATE_VERSION=${OPTARG}
+      ;;
+  esac
+done
+
+echo "Compile code"
+if [ -z "$TRAVIS_BRANCH" ]
```

Comment:
And what about `"$CIRCLECI_BRANCH"` part ?

Reply:
Yeah, it was messed up, should work now.